pages/events.html

```diff
@@ -0,0 +1,37 @@
+---
+layout: default
+title: Events
+permalink: /events
+---
+
+<div class="flex-title">
+    <h1>Events</h1>
+</div>
+
+<h2>Upcoming events</h2>
```

Comment:
Because I wanted to solve this cleanly, there is the limitation that the minute we reach the start time of the event, it becomes a past event and disappears from the home page.
There is an alternative that involves a loop over all events and an if statement to check whether the date is checked (without time), so we can keep an event on the home page for the entire day (until midnight). It's a little less clean but might be worth it. I think printing the message for 0 upcoming events will also be an ugly workaround though (counting the number of events where the if statement was true, and then afterwards checking whether that counter is still zero or not).
Let me know what you think :)